gui-apps/waybar/waybar-9999.ebuild

```diff
@@ -56,6 +57,8 @@ src_configure() {
 		$(meson_feature pulseaudio)
 		$(meson_feature tray dbusmenu-gtk)
 		$(meson_feature udev libudev)
+		$(meson_feature wifi rfkill)
+		-Dsndio=disabled
```

Comment:
@JonasToth FYI sndiod is now in tree, so we may want to test and expose this?

Reply:
Thank you very much!
I will try to take a look at it this weekend. But my time is currently low :/